man/shadow.5.xml

```diff
@@ -161,12 +158,11 @@
 	</term>
 	<listitem>
 	  <para>
-	    The number of days before a password is going to expire (see
-	    the maximum password age above) during which the user should
-	    be warned.
+	    The number of days before a password expires (see the maximum
+	    password age above) during which the user receives a warning.
```

Comment:
```
The number of days before a password expires
(see the maximum password age above)
during which the user receives a warning.
```

Reply:
I'm not convinced by `the user receives a warning`, since that's singular (one warning).  I think `the user is warned` is more appropriate.